options/glibc/include/net/if_ppp.h

```diff
@@ -0,0 +1,23 @@
+#ifndef _NET_IF_PPP_H
+#define _NET_IF_PPP_H
+
+#include <mlibc-config.h>
+
+#if defined __linux__ || defined __MLIBC_LINUX_OPTION
```

Comment:
`defined(...)` is preferable to `defined ...`

Reply:
We should write that down somewhere, as that wasn't apparent to me. I'll change it.